server/views/send/index.pug

```diff
@@ -11,6 +11,8 @@ block content
         if user.role == 'CA'
             if stage == 'PROCESSING_CA'
                 include CAtoDM.pug
+            else if licenceStatus.decisions.insufficientTimeStop
+                include CAtoDMRefusal.pug
```

Comment:
couldn't this be `if allowedTransitions.caToDmRefusal` then we don't need to pass in licenceStatus?

caToDmRefusal accounts for insufficientTimeStop and also stages (and whatever other rules may come in in future)

Allowed transitions and sending go together more tightly than individual decisions